assets/user-profile/actions.js

```diff
@@ -131,13 +131,40 @@ export function deleteTopic(topic) {
     };
 }
 
-export function shareTopic(topic) {
-    return () => {
-        return topic;
+/**
+ * Start share followed topic - display modal to pick users
+ *
+ * @return {function}
+ */
+export function shareTopic(items) {
+    return (dispatch, getState) => {
+        const user = getState().user;
+        const company = getState().company;
+        return server.get(`/companies/${company}/users`)
+            .then((users) => users.filter((u) => u._id !== user._id))
```

Comment:
afaik state.user is just id, not an object

Reply:
In `user-profile` app it is an object